ci/benchmark_parser.py

```diff
@@ -0,0 +1,159 @@
+"""
```

Comment:
Is this benchmark parser always the same ? We could think about factoring some stuff potentially

Reply:
Yes we could factor some stuff. In an ideal world, I would transfer all the parsing files to Slab repository. But it would take some time and modifications to do so.
Nevertheless, this version of the parser is a lighter one comparing to tfhe-rs. That said, tfhe-rs version could work out of the box for concrete-fft too.

Reply:
ok all good